src/Pages/Loginpage/index.js

```diff
@@ -0,0 +1,86 @@
+import React ,{Component} from 'react';
+import './style.css';
+import axios from 'axios';
+import Toastmessages from './Toastmessages';
+
+export class loginpage extends Component{
+    constructor(props){
+        super(props) 
+        this.state={
+            userName : '',
+            passWord : '',
+            loggedInMessage : '',
+            showComponent: false
+        } 
+        this.onSubmit = this.onSubmit.bind(this)
+        this.handleChange = this.handleChange.bind(this)
+    }
+    handleChange(e){
+        this.setState({ userName: this.userName.value });
+        this.setState({ passWord : this.passWord.value });
```

Comment:
Merge both two into one statement. no need to use two setState functions. It will re-render the components twice.